src/Item.js

```diff
@@ -179,9 +179,13 @@ class Item extends EventEmitter {
 
     // if the item was saved with a custom inventory hydrate it
     if (this.inventory) {
+      // handle equipped containers.
+      if (!(this.inventory instanceof Inventory)) {
```

Comment:
I thought it did too. It specifically happens when logging back onto a character who is wearing a container that has items in it. For some reason, it was not hydrating property. I should double check and make sure that this behavior isn't a bug specific to my fork :/